install_config/master_node_configuration.adoc

```diff
@@ -1148,8 +1148,14 @@ AUDIT: id="5c3b8227-4af9-4322-8a71-542231c3887b" ip="127.0.0.1" method="GET" use
 AUDIT: id="5c3b8227-4af9-4322-8a71-542231c3887b" response="200"
 ----
 
-The `openshift_master_audit_config` variable enables API service auditing. It
-takes an array of the following options:
+To enable and configure audit, edit the *_/etc/origin/master/master-config.yaml_*
+file or use a command similar to the following:
+
+----
+$ openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/lib/origin/audit-ocp.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 15}
+----
+
+Audit takes the following parameters:
```

Comment:
"Audit" is consistent with the rest of the section, but it seems like it needs to be \*Audit* or "The audit configuration."